ament_clang_format/cmake/ament_clang_format_lint_hook.cmake

```diff
@@ -0,0 +1,14 @@
+file(GLOB_RECURSE _source_files FOLLOW_SYMLINKS
+  "*.c"
+  "*.cc"
+  "*.cpp"
+  "*.cxx"
+  "*.h"
+  "*.hh"
+  "*.hpp"
+  "*.hxx"
+)
```

Comment:
I was just imagining someone causing this hook to happen while in a sub-`CMakeLists.txt`, like one in the `test` folder. I thought the glob would be relative to the `CMAKE_CURRENT_LIST_DIR` instead.